lib/cfg.py

```diff
@@ -251,6 +253,83 @@ def cfg_version(filename):
         cfg[section] = d
     return cfg
 
+def cfg_altiumWorkspace(options):
+    """
+    ricerca del nome di tutti i progetti all'interno del file Workspace
+    """
+    risultato=True
```

Comment:
Qui, sono una nota di conding style, che ho visto fare in varie parti del codice, ovvero lo spazio tra la variabile e '='.. a volte ce lo metto e a volte no.. questo è brutto.. in genere si scrive:
variablie = valore